packages/nimble/R/cppDefs_core.R

```diff
@@ -77,11 +94,14 @@ cppClassDef <- setRefClass('cppClassDef',
                                extPtrTypes = 'ANY',
                                private = 'list',		# 'list'. This field is a placeholder for future functionality.  Currently everything is generated as public
                                useGenerator = 'ANY',		#'logical',
-                               SEXPgeneratorFun = 'ANY',        ## cppFunctionDef object
-                               SEXPfinalizerFun = 'ANY'),       ## cppFunctionDef object
+                               SEXPgeneratorFun = 'ANY', ## These will be cppFunctionDefs.
+                               SEXPfinalizerFun = 'ANY',
+                               globalObjectsDefs = 'ANY'
```

Comment:
@NLMichaud Could you explain the `globalObhectsDefs` to me in person? It might be useful for the tensorflow work Perry and I are doing.